README.md

```diff
@@ -122,6 +122,27 @@ A Hugo boilerplate for creating truly epic websites. This is a boilerplate for u
 
 - [netlify-templates/victor-hugo](https://github.com/netlify-templates/victor-hugo)
 
+
+## Nuxt, TypeScript, Docker [↗](https://awesomestacks.dev/wemake-vue-template)
```

Comment:
The uri doesn't match `/wemake-vue-template`

Reply:
Please update title to `Nuxt.js, TypeScript + Docker`

uri to: `/nuxtjs-typeScript-docker`